### PR TITLE
test(j-s): Indictment E2E tests 

### PR DESCRIPTION
**Pull request by unakb**
[Skrifa e2e test fyrir ákærur - Stofna nýja ákæru og senda á dómstól](https://app.asana.com/0/1199153462262248/1206777895052515)

## What

Added test for creating a new indictment

## Why

Because we want to have this as an automatic regression test 

## Screenshots / Gifs

https://github.com/user-attachments/assets/3e711f63-9a37-42cb-80af-25a77a608692

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

## Summary by CodeRabbit

- **New Features**
  - Introduced a new test suite to validate the indictment process within the judicial system application.

- **Bug Fixes**
  - Enhanced reliability by ensuring critical functionalities related to indictment cases are properly tested and validated.

### CODE REVIEW

File: apps/system-e2e/src/tests/judicial-system/regression/indictment-tests.spec.ts [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+    // Overview
+    await expect(page).toHaveURL(`/akaera/stadfesta/${caseId}`)
+
+    await Promise.all([
+      page.getByTestId('continueButton').click(),
+      verifyRequestCompletion(page, '/api/graphql', 'TransitionCase'),
+    ])
+    await page.getByTestId('modalPrimaryButton').click()
+  })
```
[the remark this answers is not in the document]
Comment (unakb) [reply apps/system-e2e/src/tests/judicial-system/regression/indictment-tests.spec.ts:127]: I agree, but this is the format that the tests come in when they're created by using playwrights record option which we use a lot, it feels alright to keep it like this then to keep that consistency as it's what we'll continue to use. Maybe refactor at some point in the future.

File: apps/system-e2e/src/tests/judicial-system/regression/indictment-tests.spec.ts [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    await uploadDocument(
+      page,
+      async () => {
+        await page
+          .locator('section')
+          .filter({
+            hasText:
+              'Ákæra *Dragðu gögn hingað til að hlaða uppVelja gögn til að hlaða upp',
+          })
+          .locator('button')
```
Comment (AndesKrrrrrrrrrrr) [inline apps/system-e2e/src/tests/judicial-system/regression/indictment-tests.spec.ts:97]: Why not use `getByText` or `getByRole`?
```suggestion
          .getByText('Ákæra *Dragðu gögn hingað til að hlaða uppVelja gögn til að hlaða upp')
          .getByRole('button', {name: 'Ákæra *Dragðu gögn hingað til að hlaða uppVelja gögn til að hlaða upp'})
```

File: apps/system-e2e/src/tests/judicial-system/regression/indictment-tests.spec.ts [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+    await uploadDocument(
+      page,
+      async () => {
+        await page
+          .locator('section')
+          .filter({
+            hasText:
+              'Sakavottorð *Dragðu gögn hingað til að hlaða uppVelja gögn til að hlaða upp',
+          })
+          .locator('button')
```
Comment (AndesKrrrrrrrrrrr) [inline apps/system-e2e/src/tests/judicial-system/regression/indictment-tests.spec.ts:111]: ibid

**Review by AndesKrrrrrrrrrrr — APPROVED** [review]
LGTM

**Pull request merged**